subprojects/robotpy-wpilib/wpilib/iterativerobotpy.py

```diff
@@ -0,0 +1,237 @@
+import hal
+import ntcore
+import wpilib
+import wpilib._impl
+import wpilib._impl.report_error
+
+
+from enum import IntEnum
+
+
+class IterativeRobotMode(IntEnum):
+    kNone = 0
+    kDisabled = 1
+    kAutonomous = 2
+    kTeleop = 3
+    kTest = 4
+
+
+class IterativeRobotPy(wpilib.RobotBase):
```

Comment:
I know this is a draft, but adding docstrings can be annoying. Recommend either using the sphinxify webserver (https://github.com/robotpy/devtools/blob/main/sphinxify_server.py) to convert javadocs, or just grab the docstrings from the .pyi file that ships in our existing wheels on pypi.

Obviously, before we can merge this, all public methods need docstrings.